src/Provider/Provider.Web/Views/Part2/AdditionalQuestions/AdditionalQuestions.cshtml

```diff
@@ -14,22 +14,43 @@
         <partial name="@PartialNames.ValidationSummary" model='new ValidationSummaryViewModel { ModelState = ViewData.ModelState }' />
         <partial name="@PartialNames.ReviewSummary" for="Review" />
         
+        <span class="govuk-caption-l">@Model.VacancyTitle</span>
+
         <h1 class="govuk-heading-xl">
-            Do you have any questions you would like to ask applicants? (optional)
+            Application questions on Find an apprenticeship
         </h1>
-        <p class="govuk-body">Applicants will answer these questions if they apply through <a href="@Model.FindAnApprenticeshipUrl" class="govuk-body govuk-link" rel="noopener" target="_blank">Find an apprenticeship (opens in a new tab)</a>.</p>
-        <p class="govuk-body">For example, 'How will you travel to this apprenticeship?'</p>
+        <p class="govuk-body">Find an apprenticeship includes set questions that applicants will need to answer. You can add extra questions if you need to ask more.</p>
+
+        @if (Model.ApprenticeshipType == ApprenticeshipTypes.Foundation)
```

Comment:
Same as above, we can simplify this and reduce the duplication

Reply:
Amended in latest commit.